src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -25,7 +25,16 @@ class ServiceGraphPage extends React.Component<ServiceGraphProps, ServiceGraphSt
   }
 
   componentDidMount() {
-    this.setState({ elements: FakeData.getElements() });
+    API.GetGraphElements('istio-system', null) // TODO put real namespace
```

Comment:
is this todo documented somewhere outside of code? JIRA?

Reply:
Tracking in https://issues.jboss.org/browse/SWS-176